onnxruntime/core/session/onnxruntime_c_api.cc

```diff
@@ -2735,6 +2695,8 @@ static constexpr OrtApi ort_api_1_to_16 = {
     &OrtApis::GetROCMProviderOptionsAsString,
     &OrtApis::ReleaseROCMProviderOptions,
     &OrtApis::CreateAndRegisterAllocatorV2,
+
+    &OrtApis::RunAsync,
```

Comment:
```suggestion
    &OrtApis::CreateAndRegisterAllocatorV2,
    &OrtApis::RunAsync,
```

We currently don't have spaces between the functions in a specific version of the API

Reply:
Space removed.